lib/csv_row_model/model/columns.rb

```diff
@@ -20,6 +20,17 @@ def column_names
           columns.keys
         end
 
+        # @return [Array] column friendly names for the row model
+        def friendly_names
+          @friendly_names ||= begin
+            names = []
+            columns.each do |name, options|
+              names << (options[:friendly_name] || name.to_s.titleize)
```

Comment:
maybe should we move format_cell to columns? Sorry if i'm misunderstanding. But in the case of columns without friendly names i wanted  to have the name of the column instead of nil